src/components/Map/Map.js

```diff
@@ -36,10 +49,10 @@ const Map = ({ placeIds }) => {
     <LoadScript googleMapsApiKey={mapsApiKey}>
       <GoogleMap
         mapContainerStyle={containerStyle}
-        center={placeDetails.result.geometry.location}
-        zoom={12}
+        center={activeMarker ? activeMarker.position : centerCoords}
```

Comment:
Could you explain what this code is doing?  Originally we had the center set to a specific lat and long.  I am curious to see what the new code is doing and why?

Reply:
`centerCoords` has a default value of roughly the center of the US. Once locations have been fetched, the `centerCoord` is reassigned to the lat long of the first location.